src/main/java/taxi/controller/driver/AddDriverController.java

```diff
@@ -24,8 +24,10 @@ public void doGet(HttpServletRequest req, HttpServletResponse resp)
     public void doPost(HttpServletRequest req, HttpServletResponse resp) throws IOException {
         String name = req.getParameter("name");
         String licenseNumber = req.getParameter("license_number");
-        Driver driver = new Driver(name, licenseNumber);
+        String login = req.getParameter("login");
+        String password = req.getParameter("password");
+        Driver driver = new Driver(name, licenseNumber, login, password);
         driverService.create(driver);
-        resp.sendRedirect(req.getContextPath() + "/drivers/add");
```

Comment:
Here problem, sometimes you don't need to login immediately, after register new driver(maybe, you add couple of drivers), so, i recommend you return back previous logic